spec/features/admin_user/admin_change_categories_for_item_spec.rb

```diff
@@ -1,31 +1,34 @@
 require "rails_helper"
 
 RSpec.feature "admin can update categories for an item", type: :feature do
-  before(:each) do
-    dessert_cat = Category.create(name: "dessert")
-    curry_cat   = Category.new(name: "curry")
-    seafood_cat = Category.new(name: "seafood")
-    drinks_cat  = Category.new(name: "drinks")
+  let(:dessert) { Category.create(name: "Dessert") }
+  let(:seafood) { Category.create(name: "Seafood") }
+  let(:curry)   { Category.create(name: "Curry") }
+  let(:drinks)  { Category.create(name: "Drinks") }
 
-    unicorn     = Item.create(title: "Unicorn Pie",
+  let(:unicorn) { Item.create(title: "Unicorn Pie",
                               description: "Good",
                               price: 8,
                               status: "active",
-                              categories: [dessert_cat, seafood_cat])
+                              categories: [dessert, seafood]) }
 
-    elaria      = User.create(full_name: "Elaria Sand",
-                              user_name: "HouseMartell",
-                              email: "elaria@dorne.com",
-                              password: "password",
-                              role: 1)
+  let(:elaria) { User.create(full_name: "Elaria Sand",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.